content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -15,12 +15,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GATEWAY_API="${GATEWAY_API:-false}"
+
 # @setup profile=none
 
 set -e
 set -u
 set -o pipefail
 
+# Helper function to initialize KUBECONFIG_FILES and KUBE_CONTEXTS
+_set_kube_vars
+
+# Kubernetes Gateway API CRDs are required by waypoint proxy.
+source "tests/util/gateway-api.sh"
+install_gateway_api_crds "${KUBE_CONTEXTS[0]}"
+
```

Comment:
None of this should be needed if done in gtw_test.sh.
```suggestion
```

Reply:
Gateway API CRDs are needed even when using Istio Classic APIs (required by waypoint).

Reply:
Yes, I know, but my first instinct was to handle the classic case with explicit snips in the doc, which you would call instead of using the util functions.

That said, maybe fixing the util function to be usable for ambient-only, as you're doing, is a better since there will be more ambient docs coming which won't have explicit snips for the CRDs.